build openshift & k8s operator bundle container images with different tag

The rule `bundle-build` expects `manifests`, `metadata` and `tests` in `${PWD}/bundle`. But since https://github.com/os-observability/tempo-operator/pull/356 there are two variants (`openshift` and `community`). 

https://github.com/os-observability/tempo-operator/blob/5c6c1584cb28c03192ef20af6118544e54fe2933/Makefile#L214-L225

```bash
Step 12/14 : COPY bundle/manifests /manifests/
COPY failed: file not found in build context or excluded by .dockerignore: stat bundle/manifests: file does not exist
make: *** [Makefile:221: bundle-build] Fehler 1
```

--- a/Makefile
+++ b/Makefile
@@ -219,7 +219,7 @@ bundle: generate-bundle
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	docker build -f $(BUNDLE_DIR)/bundle.Dockerfile -t $(BUNDLE_IMG) $(BUNDLE_DIR)
 
 .PHONY: bundle-push
 bundle-push: ## Push the bundle image.

